sections/header.liquid

```diff
@@ -32,13 +32,10 @@
     margin-left: -1.2rem;
   }
 
-  .header__heading-logo {
-    max-width: {{ settings.logo_width }}px;
-  }
 
   {%- if section.settings.sticky_header_type == 'reduce-logo-size' -%}
     .scrolled-past-header .header__heading-logo {
-      width: 75%;
+      width: {{ settings.logo_width | times: 0.75 | round }}px;
```

Comment:
Since this doesn't reference the width from `.header__heading-logo` because we're only applying a max width on it, and instead referencing the setting. It doesn't really do anything on smaller screen when the space given to the logo itself is less than the setting's width. 

Say on mobile, the max amount of space the logo can be in is `150px`, and the `Desktop logo width` setting is at `190px` (at least `150px + (150 - 150 x 75%)`), then you end up having no `reduce logo size` effect.